packages/state-transition/src/block/processWithdrawalRequest.ts

```diff
@@ -0,0 +1,99 @@
+import {toHexString} from "@chainsafe/ssz";
+import {electra, phase0, ssz} from "@lodestar/types";
+import {
+  FAR_FUTURE_EPOCH,
+  MIN_ACTIVATION_BALANCE,
+  PENDING_PARTIAL_WITHDRAWALS_LIMIT,
+  FULL_EXIT_REQUEST_AMOUNT,
+  ForkSeq,
+} from "@lodestar/params";
+
+import {CachedBeaconStateElectra} from "../types.js";
+import {hasCompoundingWithdrawalCredential, hasExecutionWithdrawalCredential} from "../util/electra.js";
+import {getPendingBalanceToWithdraw, isActiveValidator} from "../util/validator.js";
+import {computeExitEpochAndUpdateChurn} from "../util/epoch.js";
+import {initiateValidatorExit} from "./initiateValidatorExit.js";
+
+export function processWithdrawalRequest(
+  fork: ForkSeq,
+  state: CachedBeaconStateElectra,
+  withdrawalRequest: electra.WithdrawalRequest
+): void {
+  const amount = Number(withdrawalRequest.amount);
```

Comment:
as per type, this conversion should not be required
```suggestion
  const amount = withdrawalRequest.amount;
```